target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4019-mf289f.dts

```diff
@@ -0,0 +1,433 @@
+
+                                precal_art_5000: precal@5000 {
+                                        reg = <0x5000 0x2f20>;
+                                };
+                        };
+
+                        partition@120000 {
+                                label = "mac";
+                                reg = <0x120000 0x80000>;
+                                read-only;
+                                compatible = "nvmem-cells";
+                                #address-cells = <1>;
+                                #size-cells = <1>;
+
+                                macaddr_config_0: macaddr@0 {
+                                        reg = <0x0 0x6>;
+                                };
+                        };
+
+                        partition@1a0000 {
+                                label = "reserved2";
+                                reg = <0x1a0000 0xc0000>;
+                                read-only;
+                        };
+
+                        partition@260000 {
+                                label = "cfg-param";
+                                reg = <0x260000 0x400000>;
+                                read-only;
+                        };
+
+                        partition@660000 {
+                                label = "log";
+                                reg = <0x660000 0x400000>;
+                        };
+
+                        partition@a60000 {
+                                label = "oops";
+                                reg = <0xa60000 0xa0000>;
+                        };
+
+                        partition@b00000 {
+                                label = "reserved3";
+                                reg = <0xb00000 0x500000>;
+                                read-only;
+                        };
+
+                        partition@1000000 {
+                                label = "web";
+                                reg = <0x1000000 0x800000>;
+                        };
+
+                        partition@1800000 {
+                                label = "rootfs";
+                                reg = <0x1800000 0x1d00000>;
+                        };
+
+                        partition@3500000 {
+                                label = "data";
+                                reg = <0x3500000 0x1900000>;
+                        };
+
+                        partition@4e00000 {
+                                label = "fota";
+                                reg = <0x4e00000 0x3200000>;
+                        };
+                };
+        };
+
+};
+
+&blsp1_uart1 {
+        pinctrl-0 = <&serial_pins>;
+        pinctrl-names = "default";
+        status = "okay";
+};
+
+&cryptobam {
+        status = "okay";
+};
+
+&gmac0 {
+        nvmem-cell-names = "mac-address";
+        nvmem-cells = <&macaddr_config_0>;
+};
+
+&gmac1 {
+        nvmem-cell-names = "mac-address";
+        nvmem-cells = <&macaddr_config_0>;
+        mac-address-increment = <1>;

```

Comment:
Is the MAC address offset assignment the same as in factory assignment used by MF286D?

Reply:
Yes, same partition layout as MF286D.
I'll fix the dts for mac-increment, it was the wrong one.. 😅